### PR TITLE
WIP: add fakerjs for user data, store sessionId on redis with users results

### PR DESCRIPTION
**Pull request by larrymiami**
### What does this PR do?
This PR makes use of faker.js to generate mock data.

## Checklist

- [x] All the current data types are generated by faker
- [x] Data consistency in a particular session as discussed in #29 
- [ ] All tests pass

@StanleyMasinde @Donstesh @cyrus43, here's a skeleton of what we discussed on faker and redis.In this PR, I've connected to redis and requested < 10 users from faker and stored them in redis.We can discuss more on this implementation before i resume.
Screenshot (46)

### CODE REVIEW

**Comment by StanleyMasinde** [comment]
Hi @larrymiami Thank you for taking lead on this.

**Comment by larrymiami** [comment]
@StanleyMasinde I'm almost done with this but I'm having trouble with the second test, I keep getting "cannot read properties of null" even from a simple check like return status ok.  Do you have any idea why this is happening?

**Comment by StanleyMasinde** [comment]
@larrymiami I'll check it later though. Saahi niko help up.

File: lib/redis.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const Redis = require("ioredis");
```
Comment (StanleyMasinde) [inline lib/redis.js:1]: Just curious, why did you pick ioredis instead of https://www.npmjs.com/package/redis?

File: lib/redis.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const redisUrl =
+  process.env.NODE_ENV === "production"
```
Comment (StanleyMasinde) [inline lib/redis.js:4]: I think a switch statement would be cleaner here?

File: package.json [hunk 3]
```diff
@@ -13,9 +13,13 @@
     "debug": "~2.6.9",
     "express": "~4.17.3",
     "faker": "^5.5.3",
-    "morgan": "~1.9.1"
```
Comment (StanleyMasinde) [inline package.json:16]: Why was morgan removed?

**Review by StanleyMasinde — COMMENTED** [review]
Let me run it and give feedback

File: routes/users.js [hunk 4]
```diff
@@ -1,25 +1,55 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const fs = require('fs')
-const users = JSON.parse(fs.readFileSync('database/users.json', 'utf-8'))
+const { uuid } = require("uuidv4");
+const { faker } = require("@faker-js/faker");
+
+const fs = require("fs");
+const users = JSON.parse(fs.readFileSync("database/users.json", "utf-8"));
```
Comment (StanleyMasinde) [inline routes/users.js:7]: This is the old approach and should be removed

**Comment by StanleyMasinde** [comment]
@miami78 It now works but the users are being added to the session on every visit. We need to only do this once a day

- [ ] Generate session data only once per day
- [ ] Clean up useless code
- [ ] Use faker and session on the other areas.

It should be easier from here. Tell me if you need more support.

**Review by Donstesh — APPROVED** [review]

**Comment by StanleyMasinde** [comment]
@Donstesh this PR is not ready for review

**Pull request closed without merging**